### ISSUE TITLE
 [securedrop-proxy] Update pyyaml when 4.2 release is available

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop-client#1711]
## Description

We got a GitHub security alert regarding a CVE in pyyaml: [CVE-2017-18342](https://nvd.nist.gov/vuln/detail/CVE-2017-18342). The fix, [merged in the pyyaml repo](https://github.com/yaml/pyyaml/pull/74), renames `safe_load` to `load` and `load` to `danger_load`. We're not using `safe_load` for this project, which we should do in the beta release if pyyaml 4.2 is not ready. This is acceptable for now because:

* The only YAML load call in this project is for the [proxy config](https://github.com/freedomofpress/securedrop-proxy/blob/df6490d28c67f56584968fedf0a506f333785eed/securedrop_proxy/config.py#L18), which is generated via the workstation provisioning code running in dom0. 
* This project is in alpha stage and is not used in any production environments.

We shouldn't update to latest yet due to the fact that the only 4.2* versions available on PyPI are marked as pre-release - see https://github.com/yaml/pyyaml/issues/193 for the full details. 

tl;dr We should wait until pyyaml 4.2 is officially released and then update.
